test/unit/autocomplete.ts

```diff
@@ -3,37 +3,46 @@ import * as sinon from 'sinon';
 import Autocomplete from '../../src/autocomplete';
 import suite from './_suite';
 
-const CATEOGORY = 'brand';
-const CATEOGORY_VALUES = ['a', 'b', 'c'];
+const CATEGORY = 'brand';
+const CATEGORY_VALUES = ['a', 'b', 'c'];
 const SUGGESTIONS = ['d', 'e', 'f'];
 const NAVIGATIONS = ['g', 'h', 'i'];
-const STATE = { h: 'j' };
 
 suite('Autocomplete', ({ expect, spy, stub }) => {
   let autocomplete: Autocomplete;
-  let autocompleteSuggestionsSelector: sinon.SinonStub;
-  let autocompleteCategoryFieldSelector: sinon.SinonStub;
-  let autocompleteCategoryValuesSelector: sinon.SinonStub;
-  let autocompleteNavigationsSelector: sinon.SinonStub;
+  let select: sinon.SinonSpy;
 
   beforeEach(() => {
-    Autocomplete.prototype.flux = <any>{ store: { getState: () => STATE } };
-    autocompleteSuggestionsSelector = stub(Selectors, 'autocompleteSuggestions').returns(SUGGESTIONS);
-    autocompleteCategoryFieldSelector = stub(Selectors, 'autocompleteCategoryField').returns(CATEOGORY);
-    autocompleteCategoryValuesSelector = stub(Selectors, 'autocompleteCategoryValues').returns(CATEOGORY_VALUES);
-    autocompleteNavigationsSelector = stub(Selectors, 'autocompleteNavigations').returns(NAVIGATIONS);
+    Autocomplete.prototype.flux = <any>{};
+    Autocomplete.prototype.select = spy((selector) => {
```

Comment:
I'd make this a stub instead, that way you can match the parameter and return more declaratively
```
select = Autocomplete.prototype.select = stub();
select.withArgs(Selectors.autocompleteSuggestions).returns(SUGGESTIONS);
// ...
```